src/Features/ERDDAP/Platform/Forecasts/Page/story-data.ts

```diff
@@ -0,0 +1,25 @@
+import { platform } from "stories/platform"
+
+import { forecastToStandardNames } from "./index"
+
+export const forecast_type = "wave_height"
+
+export const standard_name = forecastToStandardNames[forecast_type]
```

Comment:
Codacy found an issue: [Variable Assigned to Object Injection Sink](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6071156)